components/wmtk_components/extract_subset/internal/extract_subset_2d.cpp

```diff
@@ -62,23 +62,34 @@ wmtk::TriMesh extract_subset_2d(
 
 
 // Note: the above is a draft version of the algo, implemented in bad, drafted data structure
-// The following is new code to be finished, commented out to compile
+// The following is new code to be finished
 wmtk::TriMesh extract_subset_2d(std::vector<wmtk::Tuple> vertices, std::vector<wmtk::Tuple> triangles, std::vector<size_t> tag){
```

Comment:
I don't really understand the parameters here. Why `vertices` and `triangles`? I think you should have a `Mesh` as input in combination with an `Attribute` that is used as tag. 

Minor comment:
The `return` value should also just be a `Mesh`. This code can be written in a generic enough way so that it works for extracting triangles or just a bunch of edges. However, we can do the generalization as soon as the basic concept is there, so don't worry about that for now.